locale/en/manageiq.po

```diff
@@ -31268,9 +31230,6 @@ msgstr ""
 msgid "MiqRequest|Stamped on"
 msgstr ""
 
-msgid "MiqRequest|State"
```

Comment:
@kbrock @Fryguy the 4 above are dropped because they're no longer virtual attributes either exposed through aliased attributes if they're actual attributes or methods if they're association methods.  I think it's ok but wanted to see if you thought it could be a problem. See:
https://github.com/ManageIQ/manageiq/commit/c76f6527da73bdad00883ae3f0fe4f5d907876a2#diff-abe7a3d160c85416f97424b7300d05ad70e208791e9c15ab73ff7bf5facc9a26L20

Reply:
Merging so we can get these through to translators.  We can fix any if we find problems with it.